app/src/main/res/layout/exploration_activity.xml

```diff
@@ -53,8 +53,18 @@
             android:layout_gravity="end"
             android:contentDescription="@{viewModel.isAudioStreamingOn ? @string/audio_player_on : @string/audio_player_off}"
             android:scaleType="center"
-            app:srcCompat="@{viewModel.isAudioStreamingOn ? @drawable/ic_baseline_headset_24 : @drawable/ic_baseline_headset_off_24}"
-            android:visibility="@{viewModel.showAudioButton ? View.VISIBLE : View.GONE}" />
+            android:visibility="@{viewModel.showAudioButton ? View.VISIBLE : View.GONE}"
+            app:srcCompat="@{viewModel.isAudioStreamingOn ? @drawable/ic_baseline_headset_24 : @drawable/ic_baseline_headset_off_24}" />
+
+          <ImageButton
+            android:id="@+id/action_bottom_sheet_options_menu"
+            style="?android:actionOverflowButtonStyle"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
```

Comment:
Does it need contentDescription for screen-readers?

Reply:
By default it has content description `Button, More options` so I don't think we need to add contentDescription explicitly.